PatientWebApplication/UserMicroserviceApi/Model/ManagerUser.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+
+namespace UserMicroserviceApi.Model
+{
+    public class ManagerUser : EmployeeUser
+    {
+        public virtual List<ManagerNotification> specialNotifications { get; set; }
```

Comment:
Change property name to start with upper letter and apply in rest of code (remove 'this' from constructors).